src/Generators/Microsoft.Gen.ComplianceReports/Roslyn4.0/Microsoft.Gen.ComplianceReports.Roslyn4.0.csproj

```diff
@@ -11,10 +11,4 @@
     <MinCodeCoverage>99</MinCodeCoverage>
     <MinMutationScore>85</MinMutationScore>
   </PropertyGroup>
-
-  <ItemGroup>
```

Comment:
Doesn't removing this effectively defeat the reason to have Roslyn3.8 and Roslyn4.0 builds of the generators (which is to support older toolchains)

Reply:
It does, we should roll this back.